pyproject.toml

```diff
@@ -86,6 +86,8 @@ Home = "https://astronomer.io/"
 provider_info = "astro.__init__:get_provider_info"
 
 [tool.pytest.ini_options]
+minversion = "6.0"
+addopts = "--cov=src --cov-report=xml --cov-branch"
```

Comment:
@kaxil locally, we probably would instead use `--cov-report=report-missing` - how do you suggest we handle both different use cases (local x CI)?

Reply:
Yeah that should still work, for example

```
❯ nox -s test -- --cov-report=term-missing
```